pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -330,6 +331,16 @@ func (a *ofFlowAction) SendToController(reason uint8) FlowBuilder {
 	return a.builder
 }
 
+func (a *ofFlowAction) Meter(meterId uint32) FlowBuilder {
+	// TODO: remove OS detection after windows OVS support OF meter.
+	// Skip apply OF meter for windows ovs for now.
+	if runtime.GOOS == "windows" {
```

Comment:
we typically put these conditional statements in pipeline.go (see other examples there). IMO it makes more sense than "hiding" the fact that the `meter` action is a no-op on Windows in the implementation, as is the case here.

Reply:
Done.